concourse/scripts/ssh.sh

```diff
@@ -1,16 +1,51 @@
 #!/bin/bash
 
 set -euo pipefail
-key=/tmp/concourse_id_rsa.$RANDOM
-trap 'rm -f $key' EXIT
+TUNNEL=${1:-}
+SOCKET_DIR=~/.ssh
+SOCKET_DEF=%r@%h:%p
+SOCKET=$SOCKET_DIR/$SOCKET_DEF
 
-eval "$(make dev showenv | grep CONCOURSE_IP=)"
+download_key() {
+  key=/tmp/concourse_id_rsa.$RANDOM
+  trap 'rm -f $key' EXIT
 
-aws s3 cp "s3://${DEPLOY_ENV}-state/concourse_id_rsa" $key && chmod 400 $key
+  eval "$(make dev showenv | grep CONCOURSE_IP=)"
 
-echo
-aws s3 cp "s3://${DEPLOY_ENV}-state/generated-concourse-secrets.yml" - | \
-ruby -ryaml -e 'puts "\Sudo password is " + YAML.load(STDIN)["secrets"]["concourse_vcap_password_orig"]' 
-echo
+  aws s3 cp "s3://${DEPLOY_ENV}-state/concourse_id_rsa" $key && chmod 400 $key
+}
 
-ssh -i $key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null vcap@"$CONCOURSE_IP"
+ssh_concourse() {
+  echo
+  aws s3 cp "s3://${DEPLOY_ENV}-state/generated-concourse-secrets.yml" - | \
+  ruby -ryaml -e 'puts "\Sudo password is " + YAML.load(STDIN)["secrets"]["concourse_vcap_password_orig"]'
```

Comment:
There's no need for \ before S in '\Sudo password is'. Not sure why I had it there, but given that you have a beautifying commit in this PR, we can also beautify and unconfuse this.